Mathlib/AlgebraicTopology/Quasicategory/StrictBicategory.lean

```diff
@@ -0,0 +1,89 @@
+/-
+Copyright (c) 2025 Emily Riehl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Emily Riehl
+-/
+
+import Mathlib.CategoryTheory.Bicategory.CatEnriched
+import Mathlib.AlgebraicTopology.Quasicategory.Basic
+import Mathlib.AlgebraicTopology.SimplicialCategory.Basic
+import Mathlib.AlgebraicTopology.SimplicialSet.NerveAdjunction
+
+/-!
+# The strict bicategory of quasicategories
+
+In this file we define a strict bicategory whose objects are quasicategories,
+whose morphisms are maps of simplicial sets, and whose morphisms are homotopy classes of homotopies
+between them, defined using the simplicial interval `Δ[1]`.
+
+This is defined by transfering the simplicial enrichment of the category of quasicategories to
+a categorical enrichment by applying `hoFunctor : SSet ⥤ Cat` to the hom-objects.
+
+This strict bicategory serves as a setting to develop the formal category theory of quasicategories.
+
+## References
+
+* Emily Riehl and Dominic Verity, Elements of ∞-Category Theory
+* Emily Riehl and Dominic Verity, The 2-category theory of quasi-categories
```

Comment:
You should add these sources to references.bib, and refer to their identifiers here. This will create a link to the bibliography in the docs. For an example of how this is done, take a look at e.g. how https://github.com/leanprover-community/mathlib4/blob/master/Mathlib/CategoryTheory/Monoidal/Braided/Reflection.lean#L26 refers to the source that's defined here: https://github.com/leanprover-community/mathlib4/blob/master/docs/references.bib#L1353-L1365